go/interpreter/lfvm/instructions_test.go

```diff
@@ -1192,6 +1192,104 @@ func TestGenericCreate_MaxInitCodeSizeIsNotCheckedBeforeShanghai(t *testing.T) {
 	}
 }
 
+func TestGenericCreate_ReportsErrors(t *testing.T) {
+
+	u64Overflow := new(uint256.Int).Add(uint256.NewInt(math.MaxUint64), uint256.NewInt(math.MaxUint64))
+	one := uint256.NewInt(1)
+
+	tests := map[string]struct {
+		offset, size  uint256.Int
+		kind          tosca.CallKind
+		revision      tosca.Revision
+		expectedError error
+	}{
+		"offset overflow": {
+			offset:        *u64Overflow,
+			size:          *one,
+			expectedError: errOverflow,
+		},
```

Comment:
Something that should be tested: if the offset is larger the MaxUint64 but the size is 0, everything should be fine -- no error produced.